lib/templates/bootstrap/textbox.js

```diff
@@ -1,5 +1,6 @@
 var React = require('react');
 var { View, Text, TextInput } = require('react-native');
+import TextField from 'react-native-md-textinput';
```

Comment:
You cant' submit both commits if one is not intended to enter in the PR. Please cherry-pick your commits or open another PR. And don't forget to tell us _why_ we should merge it and _what_ it actually solves.